inc/class-checksum-verifier.php

```diff
@@ -305,4 +301,27 @@ public static function uninstall_hook() {
 			)
 		);
 	}
+
+	/**
+	 * Add deprecation notice in WP Admin.
+	 */
+	public static function add_deprecation_notice() {
+		?>
+		<div class="notice notice-warning">
+			<p><?php esc_html_e( 'Checksum Verifier is deprecated. Its functionality is integrated into AntiVirus now.', 'checksum-verifier' ); ?></p>
+			<p><?php esc_html_e( 'Switch to AntiVirus now:', 'checksum-verifier ' ); ?></p>
+			<ol>
+				<li><?php esc_html_e( 'Install and activate AntiVirus 1.4.0+.', 'checksum-verifier' ); ?></li>
```

Comment:
I'd suggest, we add a link in this step, referencing `/wp-admin/plugin-install.php?s=pluginkollektiv&tab=search&type=author` so the user gets the right plugins.

Reply:
Just wanted to type the same suggestion.
The settings page link works fine, if AntiVirus is already installed, but obviously points to nothing otherwise. Linkint the plugin setup would be convenient here.

Reply:
If added the link to the installation page (shown if AntiVirus is not active). The link to the settings page is shown only if AntiVirus is active - this was intended before, but there was a bug in my code which is fixed now.